### DISCUSSION TITLE
Files are now too big to display?

### DISCUSSION THREAD

**Discussion by nzmagpie** [owid/covid-19-data#2809] [General]
A couple of months ago I was able to download the weekly projected death data for my country of New Zealand. Now that file, on gihub, has either disappeared, or is part of a file that is too big to show at this time.  Is it still accessible at all? The file was a CSV and contained the projected weekly deaths based on the regression method of K&K from the 2015-2019 actual death data.

**Comment by lucasrodes** [comment]
Hi @nzmagpie,
Thanks for your message, and apologies for my belated response!

To assist you, could you share the path of the file you used?

We currently report and publish excess mortality data in [this folder](https://github.com/owid/covid-19-data/blob/master/public/data/excess_mortality/excess_mortality.csv)
